exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,110 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. 
+But how is the level of complexity of a given text assessed, 
+and how can we make programs which do that? In this exercise, 
+you will write a program which will determine how difficult a 
+text is to read and what age it is best suited for.
+
+We will use the [Automated readability index (ARI)][wiki] formula 
+for calculating text complexity. The ARI was developed in 1967 
+and assesses the U.S. grade level required to read a piece of text. 
+It does this by measuring the complexity of words and sentences. 
+Complexity is measured as characters per word and words per sentence. 
+A higher value is more complex.
+
+## Text Preview
+
+Your program should display the first three sentences of the text, 
+but no more than 50 characters. If the first three sentences exceed 
+50 characters, truncate them and add an ellipsis at the end. Otherwise, 
+if the text is more than three sentences, put an ellipsis on the 
+fourth line.
+
+Suppose there is a file named "ari.txt" with the following contents:
+
+> ARI was designed for military use in 1967. It was for real-time 
+> monitoring of readability on typewriters. It offered an automated 
+> counting method. It has similar results to other frameworks.
+> This validates its accuracy.
+
+Then the program should print:
+
+```text
+The text is:
+ARI was designed for military use in 1967.
+It was for real-time monitoring of readability on ...
+It offered an automated counting method.
+...
+```
+
+## Text Statistics
+
+The program should calculate and print the number of characters, words
+and sentences in the text. Sentences can end with a full stop, 
+an exclamation mark, or a question mark. The last sentence may end with 
+or without a punctuation mark. Whitespaces separate words in a sentence. 
```

Comment:
```suggestion
an exclamation mark, or a question mark. 
Whitespaces separate words in a sentence. 
```
No tests are missing the trailing puncutation. No need to add the extra complexity to user's solutions.